packages/account/src/Sections/Security/ConnectedApps/__tests__/data-table-template.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import DataTableTemplate from '../data-table-template';
+
+describe('DataTableTemplate', () => {
+    it("should render the 'DataTableTemplate' component with correct details", () => {
+        const mock_props: React.ComponentProps<typeof DataTableTemplate> = {
+            connected_apps: [
+                {
+                    app_id: 99,
+                    app_markup_percentage: 1,
+                    last_used: '2021-10-31 06:49:52',
+                    name: 'NAME',
+                    official: 0,
+                    scopes: ['read', 'admin'],
+                    appstore: null,
+                    github: null,
+                    googleplay: null,
+                    homepage: null,
+                    redirect_uri: '',
+                    verification_uri: null,
+                },
+            ],
+            handleToggleModal: () => undefined,
+        };
+        const mock_permissions = mock_props.connected_apps[0]?.scopes
+            ?.map(scope => scope.charAt(0).toUpperCase().concat(scope.substring(1)))
+            .join(', ');
```

Comment:
What is this for?

Reply:
this is fixed. used the getConnectedAppsScopes utility instead.